### DISCUSSION TITLE
Immich, unraid and database arghh

### DISCUSSION THREAD

**Discussion by Bobski2022** [immich-app/immich#9775] [Q&A] [answered]
Hi, I’m trying to get immich working and I’m unsure as to what I’ve done wrong.
This is what is happening…
This just keeps on repeating.
I’ve loaded through the apps (unraid) postgresql_immich and immich shortly after. I’ve also tried following guides from other site and for some reason I just kept getting database errors. Which I think this is the same thing…
Any help would be appreciated 
Thanks
Dave

**Comment by Bobski2022** [comment]
![IMG_1208](https://github.com/immich-app/immich/assets/113545157/8077e092-0e53-480e-905a-0a5e49d66858)
Sorry forgot the picture

**Comment by alextran1502** [comment]
Can you try using this guide? https://youtu.be/W63WpqoJD6w?si=jqn8eiGYqNJwCJKv

**Reply by MithicalM** [reply]
The link you provided it an ad for GoDaddy, do you mind re-linking it please?

**Reply by mmomjian** [reply, marked as answer]
I think he meant this video: https://www.youtube.com/watch?v=LtNWxxM5Mzg

The main issue here is probably that you didn't do this: 
- Set DOCKER_MODS=imagegenius/mods:universal-redis
- Configure REDIS_HOSTNAME to localhost

Source: https://github.com/imagegenius/docker-immich/

**Reply by Bobski2022** [reply]
Wow, this worked a treat! The DOCKER_MODS was correct but as for the host name, I had it as 192.x.x.x not localhost. Put that in and it’s working great now. 
thank you mmomjian
